src/draw/list_view.rs

```diff
@@ -67,9 +67,9 @@ where
         let icon_size = self.params.icon_size * scale;
         let icon_spacing = self.params.icon_spacing * f32::from(scale);
 
-        let top_offset = point.y + margin.top;
-        let font_size = f32::from(self.params.font_size * scale);
         let icon_size_f32 = f32::from(icon_size);
+        let font_size = f32::from(self.params.font_size * scale);
+        let top_offset = point.y + margin.top + f32::abs(f32::min(font_size - icon_size_f32, 0.)) / 2.;
```

Comment:
nitpick: seems `abs` could be removed:

```suggestion
        let top_offset = point.y + margin.top + (icon_size_f32 - font_size).max(0.) / 2.;
```